codecov.yml

```diff
@@ -0,0 +1,11 @@
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  precision: 2
+  round: up
+  range: "70...100"
+  status:
+    patch:
+      default:
+        enabled: no
```

Comment:
If it is related to #107 could you please link it in a tag?

Reply:
No, there is no relation between `codecov.yml` and #107.
This file bypass codecov path diff error :

![image](https://user-images.githubusercontent.com/7515099/76125641-7feae600-6012-11ea-85b6-e00291be24c6.png)